co2calculator/degree_days.py

```diff
@@ -0,0 +1,108 @@
+import numpy as np
+import os
+import openrouteservice
+import xarray as xr
+from datetime import timedelta
+from datetime import date as ddate
+
+#TODO: Is this the correct way to load this API Key for openrouteservice??
+# Load environment vars (TODO: Use pydantic.BaseSettings)
+ORS_API_KEY = os.environ.get("ORS_API_KEY")
+
+def get_temp_series(location: str, date_str: str):
+    """
+    Given a location returns an xarray dataset of hourly temperatures in Kelvin
+    INPUTS:
+        location (str): ex. "Bergheimer Straße 116, 69115 Heidelberg, Germany"
+        date_str (str): should be 'year-month', ex. '2020-02'
+    OUTPUTS:
+        ds (xarray dataset): has coordinates [longitude (float32), latitude (float32), time (datetime64)]
+        
+    TODO: look first at a lookup table first rather than call openrouteservice every time?
+    TODO: here pulls from data set "temp_data.nc", need to generalize, or store entire data set somewhere
+    """
+    # get latitude and longitude coordinates for location
+    clnt = openrouteservice.Client(key=ORS_API_KEY)
+    call = openrouteservice.geocode.pelias_search(clnt, location)
+    for feature in call["features"]:
+        geom = feature["geometry"]["coordinates"]
```

Comment:
Hannah comment: replace with a call to gocoding() or geocoding_structured() from distances.py?

Reply:
Second Hannah comment: Can the user also put in known coordinates directly?